packages/rx_bloc_list/pubspec.yaml

```diff
@@ -0,0 +1,37 @@
+name: rx_bloc_list
+description: A new Flutter application.
```

Comment:
@DDavidPrime, could you please fill the description

Reply:
Fixed!